### ISSUE TITLE
"KeyNotFoundException: The given key was not present in the dictionary" occurs on save load

### ISSUE THREAD

**Issue by keithlaurent** [sway2020/FindIt2#17]
Getting the following error on loading a save. Can't seem to open Find It panel now.

```
The Mod C:\Program Files (x86)\Steam\steamapps\workshop\content\255710\2133885971 [FindIt.dll] has caused an error [ModException]

Details:
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[System.UInt64,System.UInt64].get_Item (UInt64 key) [0x00000] in <filename unknown>:0 
  at FindIt.AssetTagList.Init () [0x00000] in <filename unknown>:0 
  at FindIt.FindItLoader.OnLevelLoaded (LoadMode mode) [0x00000] in <filename unknown>:0 
  at LoadingWrapper.OnLevelLoaded (UpdateMode mode) [0x00000] in <filename unknown>:0 
```

**Comment by sway2020** [comment]
@kbobthebest Thank you for the report. I think I might know where the problem is. I'll update the mod later today.

**Comment by sway2020** [comment]
@kbobthebest I've made a hot fix. Can you help me check if the bug still exists? You might need to restart Steam so it will download and update the mod. If the problem is fixed, may I know if you have any workshop assets that you copied from the workshop folder, or downloaded from somewhere else, and currently stored in your local asset folder?

**Comment by keithlaurent** [comment]
Thanks @sway2020! Will check now, but fairly certain all my assets are from the workshop.

**Comment by keithlaurent** [comment]
Thanks, the error no longer appears and I can see the search icon and panel.

**Comment by sway2020** [comment]
Fixed in 112d462849669c4276e2cdaf3f6a3955099806cd. Issue closed.
